README.md

```diff
@@ -2,10 +2,23 @@
 
 This project exports Logilica data to add it to a weekly report. 
 
-The data is obtained from team dashbaords, one dashboard per team. The list of dashboards is stored in the team_dashbaords.txt file. As teams are onboarded a new team dashboard is created in Logilica and the team_dashboards.txt file is updated with the new team's dashboard name.
+Environment variables must be set for LOGILICA_DOMAIN, LOGILICA_EMAIL, and
+LOGILICA_PASSWORD, the corresponding values can be obtained from Bitwarden.
 
-The download_pdf.py script uses Playwright, see https://playwright.dev/python/docs/intro for instructions to install Playwright. 
+The data is obtained from team dashboards, one dashboard per team. Teams data 
+is stored in the weekly_report.yaml file. 
```

Comment:
"is stored" (and "Teams data") are somewhat vague terms -- it's not obvious whether this is input for or output of the tool.

Perhaps something like, "The list of teams is read from the local `weekly_report.yaml` file."

Also, I think it would be clearer if you moved the lines about environment variables down near the instructions for running the tool (e.g., line 17).